shell/debug.sh

```diff
@@ -83,11 +83,29 @@ if [[ $HEADLESS == "true" ]]; then
   )
 fi
 
+function ctrl_c_trap() {
+  echo "killing delve with PID: $DLV_PID"
+  kill "$DLV_PID"
+  exit
```

Comment:
nit: I would explicitly exit with code 0

Reply:
I think we'd actually want `exit 0`, since [by default](https://bash.cyberciti.biz/guide/Exit_command) it'll be the exit code of the `kill` right now.